genai-perf/genai_perf/export_data/csv_exporter.py

```diff
@@ -73,6 +96,11 @@ def export(self) -> None:
             self._write_request_metrics(writer)
             writer.writerow([])
             self._write_system_metrics(writer)
+            if self._telemetry_stats is not None:
```

Comment:
You may want to put this into its own function. We want to keep the telemetry code as contained as possible so that it doesn't start sprawling throughout the code base and unrelated code.